packages/kbn-i18n/BUILD.bazel

```diff
@@ -102,38 +102,16 @@ ts_project(
   allow_js = True,
   declaration = False,
   incremental = True,
-  out_dir = "web",
+  out_dir = "target_web",
   source_map = True,
   root_dir = "src",
   tsconfig = ":tsconfig_browser",
```

Comment:
Could you extend `tsconfig.browser.json` with `types: []` to override unnecessary `node` and `test` types from `tsconfig.base.json`?
https://github.com/elastic/kibana/blob/19b8fe0464c7df8f198f1c004fc4d8d69a16f75d/tsconfig.base.json#L52-L58

Reply:
👍 we need at least the `node` ones as the code base to generate the node and web targets is the same same.